dev_guide/builds.adoc

```diff
@@ -82,41 +83,44 @@ spec:
         secret: "secret101"
     -
       type: "ImageChange"
-  source: <3>
+  source: <4>
     type: "Git"
     git:
       uri: "git://github.com/openshift/ruby-hello-world.git"
     dockerfile: "FROM openshift/ruby-22-centos7\nUSER example"
-  strategy: <4>
+  strategy: <5>
     type: "Source"
     sourceStrategy:
       from:
         kind: "ImageStreamTag"
         name: "ruby-20-centos7:latest"
-  output: <5>
+  output: <6>
     to:
       kind: "ImageStreamTag"
       name: "origin-ruby-sample:latest"
-  postCommit: <6>
+  postCommit: <7>
       script: "bundle exec rake test"
 ----
 
 <1> This specification will create a new `*BuildConfig*` named
 *ruby-sample-build*.
-<2> You can specify a list of link:#build-triggers[triggers], which cause a new
+<2> The `*runPolicy*` field controls how the builds created from this build
+configuration should run. The default value is `*Serial*`, which means new builds
+will run in serial order.
```

Comment:
slight rephrasing:

The \`\*runPolicy\*\` field controls whether builds created from this build
configuration can be run simultaneously. The default value is `*Serial*`, which means new builds
will run in serially, not simultaneously.

(yeah i know it kinda ignores SerialLatestOnly, but i don't think we need to try to explain that in the footnote here)